backend-lib/build.gradle.kts

```diff
@@ -0,0 +1,130 @@
+plugins {
+    id("com.android.library")
+    id("org.jetbrains.kotlin.android")
+    id("zcash-sdk.android-conventions")
+
+    id("org.jetbrains.dokka")
+    id("org.mozilla.rust-android-gradle.rust-android")
+
+    id("wtf.emulator.gradle")
+    id("zcash-sdk.emulator-wtf-conventions")
+
+    id("maven-publish")
+    id("signing")
+    id("zcash-sdk.publishing-conventions")
+}
+
+// Publishing information
+
+val myVersion = project.property("LIBRARY_VERSION").toString()
+val myArtifactId = "zcash-android-backend"
+publishing {
+    publications {
+        publications.withType<MavenPublication>().all {
+            artifactId = myArtifactId
+        }
+    }
+}
+
+android {
+    namespace = "cash.z.ecc.android.backend"
+
+    useLibrary("android.test.runner")
+
+    defaultConfig {
+        consumerProguardFiles("proguard-consumer.txt")
+    }
+
+    buildTypes {
+        getByName("debug").apply {
+            // test builds exceed the dex limit because they pull in large test libraries
+            isMinifyEnabled = false
+        }
+        getByName("release").apply {
+            isMinifyEnabled = project.property("IS_MINIFY_SDK_ENABLED").toString().toBoolean()
+            proguardFiles.addAll(
+                listOf(
+                    getDefaultProguardFile("proguard-android-optimize.txt"),
+                    File("proguard-project.txt")
+                )
+            )
+        }
+        create("benchmark") {
+            // We provide the extra benchmark build type just for benchmarking purposes
+            initWith(buildTypes.getByName("release"))
+            matchingFallbacks += listOf("release")
+        }
+    }
+
+    lint {
+        baseline = File("lint-baseline.xml")
+    }
+}
+
+cargo {
+    module = "."
+    libname = "zcashwalletsdk"
+    targets = listOf(
+        "arm",
+        "arm64",
+        "x86",
+        "x86_64"
+    )
+    apiLevels = mapOf(
+        "arm" to 16,
+        "arm64" to 21,
+        "x86" to 16,
+        "x86_64" to 21,
+    )
+
+    profile = "release"
+    prebuiltToolchains = true
+}
+
+dependencies {
+    api(projects.lightwalletClientLib)
+
+    implementation(libs.androidx.annotation)
+    implementation(libs.androidx.appcompat)
+
+    // For direct database access
+    // TODO [#703]: Eliminate this dependency
+    // https://github.com/zcash/zcash-android-wallet-sdk/issues/703
+    implementation(libs.androidx.sqlite)
```

Comment:
These aren't being used by backend-lib currently.  We could move the database queries into backend-lib, but I think that's beyond the scope of the changes I want to make for this PR.